client/src/views/components/Product/Product.js

```diff
@@ -0,0 +1,72 @@
+import { FILLED_HEART_ICON, HEART_ICON, MESSAGE_ICON } from '../../../constants/imagePath.js';
+import { SAMPLE_THUMBNAIL } from '../../../dummy-file.js';
+import { createDOMwithSelector } from '../../../utils/createDOMwithSelector.js';
+
+export default function Product({
+  $selector,
+  infos: {
+    title,
+    town,
+    elapsedTime,
+    price,
+    reaction: { like, message },
+    isLiked,
+  },
+}) {
+  this.$Product = createDOMwithSelector('div', '.product');
+  this.$selector = $selector;
+  this.$selector.appendChild(this.$Product);
```

Comment:
this.$selector = $selector를 선언함으로써 $selector를 인스턴스가 가지게 되는데, 
$selector는 this.$Product를 자식요소에 넣기 위한 역할만 함으로써 
this (인스턴스)가 가지고 있을 필요가 없어 보입니다! 
```suggestion
  $selector.appendChild(this.$Product);
```

Reply:
그렇네요...! 조언 감사합니다~~